example/android/build.gradle

```diff
@@ -1,9 +1,12 @@
 buildscript {
-    ext.kotlin_version = '1.6.10'
+    ext.kotlin_version = '1.6.0'
```

Comment:
Why are you downgrading Kotlin? Latest is `1.8.20`, but even with `1.6.x` it should be at least `1.6.21`

Reply:
Done